### DISCUSSION TITLE
Updating from version 1.21.0 to 1.22.0

### DISCUSSION THREAD

**Discussion by soygabrielnunez** [dm4t2/vue-currency-input#225] [General]
Hello, I just did an npm install to my project which is using this library, and I got errors relating this component, because its methods/API interface changed. I understand that, when any library changes its API methods/interface, that is a major/breaking change, and hence, following the semantinc versioning (or semver for short) the new version should have been 2.0.0, not 1.22.0, since it's a breaking change, not just adding new features or fixing some bugs. Anyone who does an npm install in a project that happens to have `"vue-currency-input": "^1.21.0"` in the package.json will have this problem, because npm updates the second and third digit of the version in the package-lock.json when using the ^ character in the package current major version... to ensure it doesn't break I changed it to `"vue-currency-input": "~1.21.0"` and did an npm install again, so I can see what are those breaking changes later. I just want to point out that this can happen to anyone, so please be more careful when there are breaking changes 🙏 and thank you very much for making and maintaining this library!

**Comment by dm4t2** [comment]
Hi, thanks for your feedback. 

This was definitely my bad and there were already multiple complaints about this (see #164 and #149).

I will consider semantic versioning for future releases.

**Reply by soygabrielnunez** [reply]
Nice! Thank you 😊and keep it up!
